src/ui/Combobox/ComboboxInput.ts

```diff
@@ -73,6 +73,14 @@ export class ComboboxInput {
     this.textInput.reset();
   }
 
+  private handleFocusOut(event: FocusEvent) {
+    const newTarget = event.relatedTarget as HTMLElement;
+    if (newTarget && this.combobox.element.contains(newTarget)) {
```

Comment:
What does the case `combobox.element.contains(newTarget)` being `true` represent? Is it when the focus is inside the input? If so, it would be nicer to store it in a well named variable.

```
const isFocusedOnInput = this.combobox.element.contains(newTarget)
```


I'm also curious under which situation(s) will `newTarget` be undefined?

Reply:
Good point, I hesitated to introduce an extra variable. I'll add one.

As for `newTarget` being undefined, at-least as far as I remember, `focusout`'s `relatedTarget` is set to `undefined`, `null` or the root `<html`> tag when the browser's UI takes the focus.